content/en/sensitive_data_scanner/regular_expression_syntax.md

```diff
@@ -0,0 +1,225 @@
+---
+title: Regular Expression Syntax
+kind: documentation
+disable_toc: false
+further_reading:
+    - link: "/sensitive_data_scanner/"
+      tag: "Documentation"
+      text: "Set up Sensitive Data Scanner"
+---
+
+## Overview
+
+In [Sensitive Data Scanner][1], the scanning rule determines what sensitive information to match within the data. You can use rules from the [Scanning Rule Library][2] or you can create custom scanning rules using regular expression (regex) patterns to scan for sensitive information. The Sensitive Data Scanner regex syntax is a subset of [PCRE2][3].
+
+{{< whatsnext desc="The available regex syntax for custom rules are broken down into the following categories:" >}}
+    {{< nextlink href="sensitive_data_scanner/regular_expression_syntax#alternation" >}}Alternation{{< /nextlink >}}
+    {{< nextlink href="sensitive_data_scanner/regular_expression_syntax#assertions" >}}Assertions{{< /nextlink >}}
+    {{< nextlink href="sensitive_data_scanner/regular_expression_syntax#ascii_classes" >}}ASCII classes{{< /nextlink >}}
```

Comment:
```suggestion
    {{< nextlink href="sensitive_data_scanner/regular_expression_syntax#ascii-classes" >}}ASCII classes{{< /nextlink >}}
```

Reply:
Thanks for the catch! I don't think they plan on making more regex syntax available.